plugin/src/test/java/jenkins/plugins/openstack/compute/slaveopts/BootSourceTest.java

```diff
@@ -55,7 +55,7 @@
 import static org.mockito.Mockito.when;
 
 public class BootSourceTest {
-    private static final FormValidation VALIDATION_REQUIRED = FormValidation.error(hudson.util.Messages.FormValidation_ValidateRequired());
+    private static final FormValidation VALIDATION_REQUIRED = FormValidation.error(hudson.util.Messages._FormValidation_ValidateRequired().toString(US));
```

Comment:
If running the Test jenkins.plugins.openstack.compute.slaveopts.BootSourceTest#doCheckImageIdWhenNoValueSet at a machine without US or GB localisation the assertion will fail, because the created VALIDATION_REQUIRED which is used for the expect value changes based on the jvm default java.util.Locale (See the Implementation org.jvnet.localizer.ResourceBundleHolder#format)
This change forces the US Localisation for the creation of the expected value of VALIDATION_REQUIRED.